apps/fe-admin/src/components/Sidebar/Sidebar.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import styles from './Sidebar.module.scss';
+import { Link, UserInterface } from '../../types';
+import Dashboard from '../../assets/images/dashboard.svg';
+import Product from '../../assets/images/product.svg';
+import Order from '../../assets/images/order.svg';
+import User from '../../assets/images/user.svg';
+import Navigation from '../Navigation/Navigation';
+import UserComponent from '../User/User';
+
+const Sidebar = () => {
+  const SIDEBAR_LINKS: Link[] = [
+    {
+      name: 'Dashboard',
+      link: '/',
+      imgLink: (
+        <img src={Dashboard} alt='dashboard-icon' className={styles.icon} />
```

Comment:
```suggestion
        <DashboardIcon className={styles.icon} />
```

Reply:
Please read about https://www.npmjs.com/package/vite-plugin-svgr
It is already installed